### ISSUE TITLE
markup du shortcode toggle 

### ISSUE THREAD

**Issue by knuch** [epfl-idevelop/doc-epfl-antistatique#12]
Il semblerait que le shortcode `epfl_toggle` soit du html statique à afficher tel quel. 
Cependant, nous avons créé un composant html/css qui n'a pas le même markup. Est-ce qu'il sera possible d'adapter le rendu de ce shortcode à nos beosins?

**Comment by lboatto** [comment]
Oui on pourrait changer le markup et adapter le css.

Une question cependant, est-ce logique d'avoir certain shortcodes ayant leur rendu côté plugin, et d'autre côté thème? Ne pourrait-on pas tout mettre dans le thème, vu que vous vous chargez du rendu?

**Comment by knuch** [comment]
Non, en effect cela ne fait pas de sens de faire du rendu côté plugin. 
En définitif, tous les plugins devraient simplement mettre à dispo des données formatées, qu'on affichera ensuite avec des composants du styleguide.
